xbrowse_server/templates/family/family_home.html

```diff
@@ -145,12 +146,83 @@ <h4 class="section-header">Individuals</h4>
                         <td>{% if indiv.maternal_id %}{{ indiv.maternal_id }}{% else %}-{% endif %}</td>
                         <td>{% if indiv.paternal_id %}{{ indiv.paternal_id }}{% else %}-{% endif %}</td>
                         <td>{% for ipheno in indiv.get_phenotypes %}{{ ipheno.phenotype.name }}{% empty %}-{% endfor %}</td>
+                        <td><button id={{ indiv }}_{{ project.project_id }} type="button" class="btn btn-info btn-sm phenotipsEditModalBtn">Edit in Phenotips</button></td>
                     </tr>
+                    
+                    
                     {% endfor %}
                     </tbody>
                 </table>
             </div>
             
+
+
+        <!-- Phenotips integration -->
+		<div id="phenotipsModal" class="modal" role="dialog">
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<button type="button" class="close" data-dismiss="modal">&times;</button>
+						<h4 class="modal-title">Phenotips</h4>
+					</div>
+					<div class="modal-body"">
+						<iframe name="phenotipsEditPatientFrame" id="phenotipsEditPatientFrame" src="" width=1000 height=800 frameborder="0"></iframe>
+					</div>
+				</div>
+			</div>
+		</div>
+		<script>
+		$('.phenotipsEditModalBtn').on('click',function(event) {
+			sessionStorage.setItem('currentInFocusIndividualId',this.id)
+			var patientDetails=this.id.split('_');
+			var patientId=patientDetails[0];
+			var project=patientDetails[1];
```

Comment:
I agree, I think I was looking at the 1kg dataset and thought the nomenclature for individual names were standardized and the risk of "_" being in a name was minimum. Though after yesterdays meeting with Hayley, I am thinking it might not be as minimum. For now changed it to a "|" separator. If you feel uncomfortable about using a separated string like this, I could generate a unique number per individual, and then a local cache that maps that number to id + project for functions downstream to use. That is a bit of extra code overhead and compute time for browser, so didn't go for it as the first solution and went with the simpler separated string. Let me know if you prefer the latter strategy. For now committed the changes using a "|" separator.